onchain/src/social/account.cairo

```diff
@@ -49,9 +61,10 @@ pub mod SocialAccount {
         }
         fn handle_transfer(ref self: ContractState, request: SocialRequest<Transfer>) {
```

Comment:
`handle_transfer` should be a part of social component

Reply:
Refactored the code in my latest commit 👍